python/cuml/cuml/accel/core.py

```diff
@@ -186,14 +186,3 @@ def install(
     set_global_output_type("numpy")
 
     logger.info("Accelerator installed.")
-
-
-def pytest_load_initial_conftests(early_config, parser, args):
```

Comment:
Duplicated from `pytest_plugin.py` which seems to be the one that is actually used